src/content/self-hosted/administration/configuration.mdx

```diff
@@ -69,7 +69,7 @@ https://private-endpoints.${subdomain}
 
 ### auth
 
-Okteto supports using Bitbucket, GitHub, Google, or OpenID Connect as authentication providers.
+Okteto supports using Bitbucket, GitHub, Google, OpenID Connect, or Token-based as authentication providers.
```

Comment:
Small suggested change:

> or token-based methods as authentication

Something too add a bit of clarity here since grammatically this seems a bit odd to me 😄 "Methods" can be a different description if we like, but I'm assuming that's the intent here.